library/ssl_ciphersuites.c

```diff
@@ -1874,6 +1874,24 @@ int mbedtls_ssl_get_ciphersuite_id( const char *ciphersuite_name )
     return( cur->id );
 }
 
+#if defined(MBEDTLS_SSL_PROTO_TLS1_3)
+unsigned int mbedtls_hash_size_for_ciphersuite(const mbedtls_ssl_ciphersuite_t* ciphersuite)
+{
+    /* We assume that the input parameter, ciphersuite, is not NULL. */
+    switch( ciphersuite->mac )
+    {
+    case MBEDTLS_MD_SHA256:
+        return( 32 );
+    case MBEDTLS_MD_SHA384:
+        return( 48 );
+    case MBEDTLS_MD_SHA512:
+        return( 64 );
+    default:
+        return( 0 );
+    }
+}
+#endif /* MBEDTLS_SSL_PROTO_TLS1_3 */
```

Comment:
It can be replace with
```c
    psa_algorithm_t hash_alg = PSA_ALG_HKDF_GET_HASH( alg );
    uint8_t hash_length = PSA_HASH_LENGTH( hash_alg );
```